EIPS/eip-4973.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 4973
+title: Non-Transferrable Non-Fungible Tokens (Soulbound Tokens or "Badges")
+description: A standard interface for non-transferrable non-fungible tokens, also known as "soulbound tokens" (short "SBT") or "badges".
+author: Nicola Greco, Tim Daubenschütz (TimDaub)
+discussions-to: https://ethereum-magicians.org/t/eip-4973-non-transferrable-non-fungible-tokens-soulbound-tokens-or-badges/8825
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-01
+requires: 165, 721
+---
+
+## Abstract
+
+The following standard allows for implementing a standard API for SBTs within smart contracts. It provides basic functionality to gift, mint and track SBTs.
+
+## Motivation
+
+In various GitHub issues, pull requests, and Ethereum's governance forums, its community, has expressed a need for non-transferrable, non-fungible tokens. Sometimes those "NTTs" are referred to as "soulbound tokens" ("SBTs") in reference to popular MMORPGs such as World of Warcraft, where items, upon looting, are bound with the player's soul and hence could not subsequently be transferred to other characters anymore. Other times, when using them as credentials, NTTs have been called "badges."
```

Comment:
Extra comma, I don't think you need to add an extra initialism ("NTT") since we're already using "SBT" from the title, and I'd move the sentences about why you've chosen "Soulbound Token" to the "Rationale" section, since it is a choice specific to this EIP.

```suggestion
The Ethereum community has expressed a need for non-transferrable, non-fungible tokens. Common use cases include tracking an individual's achievements (which can be used as credentials), or items that cannot be transferred (like certain loot in multiplayer online games).
```